src/main/java/ro/developmentfactory/thedrones/entity/Drone.java

```diff
@@ -0,0 +1,37 @@
+package ro.developmentfactory.thedrones.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+public class Drone {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id_drone")
+    private Long idDrone;
+
+    @Column(name = "name")
+    private String name;
+    @Column(name = "count_move")
+    private int countMove;
+    @Column(name = "created_at")
```

Comment:
please use `columnDefinition = "TIMESTAMP"` for all your timestamps in the database, otherwise, it creates `datetime` type and it's not what we need. you will need it for all columns with time. you can read more [here](https://stackoverflow.com/questions/409286/should-i-use-the-datetime-or-timestamp-data-type-in-mysql)

Reply:
done